src/components/numbers/Numbers.js

```diff
@@ -0,0 +1,19 @@
+import './numbers.scss';
+
+const Numbers = (props) => {
+  const { from } = props;
+  const { to } = props;
+  const { even } = props;
+  const arr = [];
+
+  for (let num = from; num <= to; num++) {
+    arr.push(num);
+    num++;
+  }
+
+  return even
+    ? <ul className="numbers">{arr.filter(item => (item % 2 === 0)).map(item => <li>{item}</li>)}</ul>
+    : <ul className="numbers">{arr.filter(item => (item % 2)).map(item => <li>{item}</li>)}</ul>;
```

Comment:
you have the same `ul` for all cases, why need to duplicate it?
What about simple case when you have not odd and not even?